solve depricated imresize of scipy.misc according to issue #122

stg7 opened the pull request:
using PIL it will solve the depreacted imresize problem of latest scipy release

beyondmetis commented:
Thank you for fixing this! I'll review when the checks finish and merge in ASAP

gdavila commented:
Hi guys, To take full advantage of this merge, please could you replace the deprecated imresize in all function using it? https://github.com/scikit-video/scikit-video/search?q=imresize&unscoped_q=imresize

stg7 commented:
> Hi guys, To take full advantage of this merge, please could you replace the deprecated imresize in all function using it? https://github.com/scikit-video/scikit-video/search?q=imresize&unscoped_q=imresize

i changed the files where imresize was used, i added a separate image.py in utils, to not copy paste the same code again and again
